text/0000-ques-in-main.md

```diff
@@ -374,25 +512,15 @@ However, the stdlib should not get in the way of a program that
 intends to conform to any of the above.  This can be done with the
 following refinements:
 
-* The stdlib provides `EXIT_SUCCESS` and `EXIT_FAILURE` constants in
-  `std::process` (since `exit` is already there).  These constants do
-  _not_ necessarily have the values that the platform's C library
-  gives them.  `EXIT_SUCCESS` is always 0.  `EXIT_FAILURE` has the same
-  value as the C library gives it, _unless_ the C library gives it the
-  value 1, in which case 2 is used instead.
-
-* All of the impls of `Termination` in the stdlib are guaranteed to
-  use only `EXIT_SUCCESS` and `EXIT_FAILURE`, with one exception:
-
-* There is a type, provisionally called `TermStatus`, which is a
-  newtype over `i32`; on Unix (but not on Windows), creating one from
-  a value outside the range 0 ... 255 will panic.  It implements
-  `Termination`, passing its value to `exit` and not printing any
-  diagnostics.  Using this type, you can generate specific exit codes
-  when appropriate, without having to avoid using `?` in `main`.
+* All of the implementations of `Termination` in the stdlib, except
+  the one for `ExitStatus` itself, are guaranteed to behave as-if they
+  use only `ExitStatus::ok()` and `ExitStatus::failure()`.
 
-  (It can't be called `ExitStatus` because that name is already
-  taken for the return type of `std::process::Command::status`.)
+* The value used by `ExitStatus::failure()` is guaranteed to be
+  greater than 1 and less than 64.  This avoids collisions with all of
+  the above conventions.  (I recommend we actually use 2, because
+  people may think that any larger value has a specific meaning, and
+  then waste time trying to find out what it is.)
```

Comment:
It's a nit, but I'd suggest 1, precisely because it's the default thing people will expect given the standard definition of EXIT_FAILURE, and the default thing people will return for "no match" kinds of "shell false" conditionals.